addons/website_blog/static/src/js/tours/website_blog.js

```diff
@@ -31,6 +31,13 @@ odoo.define("website_blog.tour", function (require) {
         extra_trigger: "#wrap div[data-oe-expression=\"blog_post.name\"]:not(:containsExact(\"\"))",
         content: _t("Set a blog post <b>cover</b>."),
         position: "right",
+    }, {
+        trigger: ".o_select_media_dialog .o_we_search",
```

Comment:
No reliance on the service during testing: we don't query the library if the search field is empty. I have to trigger an input event on the search field because of FP's request that we don't show default images when you open the media-dialog, but when searching for nothing it will only query the database and not external services.